mathesar/models.py

```diff
@@ -62,8 +63,16 @@ def _sa_engine(self):
 
     @property
     def supported_types(self):
```

Comment:
Is this the moment to add support for actually checking the DB to see if the type is supported by the current mathesar installation (i.e., do the db_types actually exist in the DB)?

Or should we wait for a future issue for that?

Reply:
I think we should wait for a future issue. There's already one that should cover it:
- https://github.com/centerofci/mathesar/issues/403